resources/public/api/conf/1.0/retrieveTransactionDetails.yaml

```diff
@@ -0,0 +1,95 @@
+get:
+  summary: Retrieve a Self Assessment Transaction's Detail [test only]
+  description: |
+    This endpoint allows a developer to retrieve more detail about a specific transaction. A unique identifier must be used for both the account (National Insurance number) and the transaction (transactionId).
+    
+    ### Test data
+    <p>Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.</p>
+    <table>
+        <thead>
+            <tr>
+                <th>Header Value (Gov-Test-Scenario)</th>
+                <th>Scenario</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><p>N/A - DEFAULT</p></td>
+                <td><p>Simulate a successful response with a self-assessment accounts.</p></td>
+            <tr>
+                <td><p>PAYMENT_ON_ACCOUNT</p></td>
+                <td><p>Simulate a successful response with a payment.</p></td> 
+            <tr>
+                <td><p>NOT_FOUND</p></td>
+                <td><p>Simulates the scenario where no data is found.</p></td>
+            <tr>
+                <td><p>NO_DETAILS_FOUND</p></td>
+                <td><p>Simulate the scenario where no transaction details can be found for this transaction ID.</p></td>
+            </tr>
+        </tbody>
+    </table>
+
+  parameters:
+    - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+    - $ref: './common/pathParameters.yaml#/components/parameters/transactionId'
+    - $ref: './common/headers.yaml#/components/parameters/acceptJson'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationReadScopeHeader'
+    - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+  security:
+    - User-Restricted:
+        - read:self-assessment
+
+  responses:
+    "200":
+      description: Success
+      headers:
+        X-CorrelationId:
+          $ref: './common/headers.yaml#/components/parameters/correlationId'
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/retrieve_transaction_details_response.json'
+          examples:
+            example-1:
+              description: Payment example
+              value:
+                $ref: './examples/retrieveTransactionDetails/retrieve_payment_response.json'
+            example-2:
```

Comment:
Can we replace "example-2:" with "Charge Example:"? I think it makes the example drop-down menu easier to read